docs/get-started.md

```diff
@@ -0,0 +1,125 @@
+
+<!-- What the user needs to know and/or do before they install EDOT Python -->
+<!-- Is this missing anything? -->
+## Prerequisites
+
+Before getting started, you'll need somewhere to send the gathered OpenTelemetry data, so it can be viewed and analyzed. EDOT Python supports sending data to any OpenTelemetry protocol (OTLP) endpoint, but this guide assumes you are sending data to an [Elastic Observability](https://www.elastic.co/observability) cloud deployment. You can use an existing one or set up a new one.
+
+<details>
+<summary><strong>Expand for setup instructions</strong></summary>
+
+To create your first Elastic Observability deployment:
+
+1. Sign up for a [free Elastic Cloud trial](https://cloud.elastic.co/registration) or sign into an existing account.
+1. Go to <https://cloud.elastic.co/home>.
+1. Click **Create deployment**.
+1. When the deployment is ready, click **Open** to visit your Kibana home page (for example, `https://{DEPLOYMENT_NAME}.kb.{REGION}.cloud.es.io/app/home#/getting_started`).
+</details>
+
+<!-- ✅ How to install EDOT Python -->
+## Install
+
+### Install the distribution
+
+Install EDOT Python:
+
+```bash
+pip install elastic-opentelemetry
+```
+
+### Install the available instrumentation
+
+EDOT Python does not install any instrumentation package by default, instead it relies on the
+`opentelemetry-bootstrap` command to scan the installed packages and install the available instrumentation.
+The following command will install all the instrumentations available for libraries found installed
+in your environment:
+
+```bash
+opentelemetry-bootstrap --action=install
+```
+
+> [!NOTE]
+> Add this command every time you deploy an updated version of your application (in other words, add it to your container image build process).
+
+<!-- ✅ Start-to-finish operation -->
+## Send data to Elastic
+
+After installing EDOT Python, configure and initialize it to start sending data to Elastic.
+
+<!-- ✅ Provide _minimal_ configuration/setup -->
+### Configure EDOT Python
+
+To configure EDOT Python, at a minimum you'll need your Elastic Observability cloud deployment's OTLP endpoint and
+authorization data to set a few `OTLP_*` environment variables that will be available when running EDOT Python:
+
+* `OTEL_RESOURCE_ATTRIBUTES`: Use this to add a service name that will make it easier to recognize your application when reviewing data sent to Elastic.
+* `OTEL_EXPORTER_OTLP_ENDPOINT`: The full URL of the endpoint where data will be sent.
+* `OTEL_EXPORTER_OTLP_HEADERS`: A comma-separated list of `key=value` pairs that will
+be added to the headers of every request. This is typically used for authentication information.
+
+<!--
+These are the instructions used in other distro docs, but in the README in this repo
+it looks like you might be recommending using an API key rather than using the secret
+token method used in the setup guides in Kibana.
```

Comment:
Yeah, I mostly use serverless for development. Serverless only offers API keys.

Reply:
Ah that makes sense.

Reply:
What do you think about adding information about using API keys in a note in the [get started guide](https://github.com/colleenmcginnis/elastic-otel-python/blob/prep-docs/docs/get-started.md#configure-edot-python)? Alternatively, if we want to use the API key approach as the primary approach, what's the best way to instruct users to find the appropriate value for the `OTEL_EXPORTER_OTLP_ENDPOINT`? Still in the set up guides in Kibana? (I want to make it as easy as possible for them to copy/paste the right values for the minimal configuration.)

Reply:
> What do you think about adding information about using API keys in a note in the [get started guide](https://github.com/colleenmcginnis/elastic-otel-python/blob/prep-docs/docs/get-started.md#configure-edot-python)? Alternatively, if we want to use the API key approach as the primary approach, what's the best way to instruct users to find the appropriate value for the `OTEL_EXPORTER_OTLP_ENDPOINT`? Still in the set up guides in Kibana? (I want to make it as easy as possible for them to copy/paste the right values for the minimal configuration.)

I don't really have a preference. The real issue is that cut & paste from kibana is not  working right now because Python requires the values to be url-encoded. The next release of the sdk will have this limit lifted hopefully so I can release a distro that behaves like other languages.
As a stopgap I've added some hints in the docs (only in main) for `OTEL_EXPORTER_OTLP_HEADERS` in 
https://github.com/elastic/observability-docs/blob/main/docs/en/observability/apm/otel-direct.asciidoc#send-data-from-an-upstream-opentelemetry-sdk. Will remove these once the limit is lifted.